### PR TITLE
Update Branches to Noetic

### PR DESCRIPTION
**Pull request by DLu**
### Description

This updates many of the references to `melodic` (and sometimes `indigo`) to refer to `noetic`. 

This does *not* include references to `noetic` entities that do not exist yet, like for `moveit_graps` and `panda_moveit_config`

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/tests/tests_tutorial.rst
Comment (AndyZe) [inline doc/tests/tests_tutorial.rst:74]: Prob should change this to `sudo apt install`

**Review by AndyZe — APPROVED** [review]
Looks good! I checked that the links still work. One minor nitpick

File: doc/tests/tests_tutorial.rst
Comment (DLu) [reply doc/tests/tests_tutorial.rst:74]: I thought about making that change.

**Comment by DLu** [comment]
The CI is not passing because changing the conf updated some of the internal links, mainly for [https://github.com/ros-planning/panda_moveit_config/issues/72](Panda Moveit Config). Should I 
 * Revert the conf changes
 * Manually edit the conf to point at different panda branch
 * Wait until the branch exists to merge this

**Comment by AndyZe** [comment]
I lean toward manually editing the conf with a TODO, but all 3 options sound OK.

The format we usually use for TODOs is:  `TODO(andyz): use `ros_distro` when that branch is available again`

**Comment by AndyZe** [comment]
thanks!

**Pull request merged**
